### ISSUE TITLE
openhab2-out2 missing output channel

### ISSUE THREAD

**Issue by lsafelix75** [jeroenhendricksen/node-red-contrib-openhab3#26]
Hi, 

Consider replicate nodered-contrib-openhab2's openhab2-out with an 'output' channel.

Alter the following lines:
 ```
 77-openhab2.html - line 749 - change output: 0 --> output: 1 
 77-openhab2.js - line 707 - add: node.send(msg);
```

**Comment by jeroenhendricksen** [comment]
Thank you for this issue. I'm considering implementing this feature. 

With regards to the implementation of it, would you agree that the output channel would - in case the 'Only when checked' checkbox was set - outputs the message when a messages was actually sent to the item or group? Of course when this checkbox is not set, it will always output a message when it was successfully set.

**Comment by jeroenhendricksen** [comment]
This feature has been implemented in 1.3.32. Please let me know if it works for you.
